### DISCUSSION TITLE
Apple 设备快速输入 Obsidian Memos

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#19] [General]
# Apple 设备快速输入 Obsidian Memos

做为个人 PKM 关键的输入环节，通过捷径（shortcut）为 iPhone、iPad、Mac 快速给 Obsidian 记录 Memos

https://www.edony.ink/apple-quick-obsidian-memo/

**Comment by hinzer** [comment]
请问 Obsidian 能跨系统吗？iOS 的平板和手机能和 LinuxOS 主机之间数据同步，用 Obsidian + Git + Github 这个方案呢？

**Reply by edonyzpc** [reply]
理论上是可以的，我尝试过用 Github 方案在 macOS 和 Android 设备之间的同步，Obsidian 有 Linux 版本所以理论上是可以的。
